packages/web-react/src/components/FileUploader/FileUploaderInput.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { RefObject, createRef } from 'react';
+import classNames from 'classnames';
+import { SpiritFileUploaderInputProps } from '../../types';
+import { useStyleProps } from '../../hooks';
+import { DEFAULT_FILE_QUEUE_LIMIT, DEFAULT_FILE_SIZE_LIMIT } from './constants';
+import { useFileUploaderStyleProps } from './useFileUploaderStyleProps';
+import { useFileUploaderInput } from './useFileUploaderInput';
+import { Icon } from '../Icon';
+
+const FileUploaderInput = (props: SpiritFileUploaderInputProps) => {
+  const {
+    children,
+    fileLimitMaxSize = DEFAULT_FILE_SIZE_LIMIT,
+    fileQueueLimit = DEFAULT_FILE_QUEUE_LIMIT,
+    helperText,
+    id,
+    isDisabled,
+    isRequired,
+    label,
+    labelText,
+    linkText,
+    onError,
+    validationMessage,
+    validationState,
+    ...restProps
+  } = props;
+
+  const hasDragAndDrop = 'draggable' in document.createElement('span');
```

Comment:
Please, use the same naming as in `web`.

Reply:
done